### PR TITLE
TTN frequency plan support China 470(uplink).

### PR DESCRIPTION
**Pull request by imliubo**
Calculated based on this [link](https://www.thethingsnetwork.org/docs/lorawan/frequency-plans.html#cn470-510), I have tested it and it can be used normally in China.

Fix codestyle check faild. :)

### CODE REVIEW

**Comment by imliubo** [comment]
Why still failed :(

File: adafruit_tinylora/ttn_cn.py
Comment (brentru) [inline adafruit_tinylora/ttn_cn.py]: Please add your name here, you wrote this file :) 

or add a line underneath my name as `# Modified by IAMLIUBO, 2021`

**Review by brentru — APPROVED** [review]
Frequency mapping plans look good

**Comment by brentru** [comment]
@dherrada  Could you please check the CI, failing on a licensing issue (similar to what happened with MiniMQTT)

File: adafruit_tinylora/ttn_cn.py
Comment (imliubo) [reply adafruit_tinylora/ttn_cn.py]: Add done. :)

Comment (brentru) [reply adafruit_tinylora/ttn_cn.py]: Thanks, you deserve credit! I'll wait for advice on resolving the CI issue and then merge this in.

Comment (imliubo) [reply adafruit_tinylora/ttn_cn.py]: Haha, OK.
I thought it was a problem with my coding style, but it seems to be a problem with the CI script (I guess...
Have a nice day.
I need to rest.

**Comment by evaherrada** [comment]
@brentru as soon as #34 gets merged, I can just merge adafruit/master into imliubo/master and we should be all good

**Comment by brentru** [comment]
@dherrada  https://github.com/adafruit/Adafruit_CircuitPython_TinyLoRa/pull/34 has been merged, bumping this

**Comment by evaherrada** [comment]
@brentru Ok. Just merged adafruit/master into imliubo/master and added a license

**Pull request merged**
